src/ec/packages/ec-component-language-list/language-list.story.js

```diff
@@ -23,7 +23,10 @@ storiesOf('Components/Language list', module)
     () =>
       languageListSplash(
         merge(dataSplash, {
-          icon_path: iconPath,
+          icon_path: text('Icon path', iconPath),
```

Comment:
what the purpose of having `icon_path` as a knob?

Reply:
You can find this in the description of the ticket. To me this knob looks also a little bit to going far

Reply:
You are both right, ideally in my vision this should be a "fake" knob, a disabled text input showing the parameter and the value without giving the chance to modify it.
The reason why in this case this icon_path was mentioned as a knob is that it's usage is spread in the different component templates, but i agree that there are more disadvantages than advantages in having such a knob. Let's remove it then.